rewrite-java/src/main/java/org/openrewrite/java/tree/Space.java

```diff
@@ -282,12 +282,12 @@ public String toString() {
         ANNOTATION_ARGUMENT_SUFFIX,
         ANNOTATION_PREFIX,
         ARRAY_ACCESS_PREFIX,
-        ARRAY_DIMENSION_PREFIX,
         ARRAY_INDEX_SUFFIX,
         ARRAY_TYPE_PREFIX,
         ASSERT_PREFIX,
         ASSIGNMENT,
         ASSIGN_OP_PREFIX,
+        ASSIGN_OP_OPERATOR,
```

Comment:
We now sit at 128 distinct space locations. While not prime, it is a power of 2, which is another thing we can nerd out about.